README.md

```diff
@@ -39,14 +41,23 @@ A [Containerfile](./custom_scripts/Containerfile) is included, so users can buil
 
 You need a FakeFish process for each server you plan to install. Think of FakeFish like if it was a custom implementation of a BMC for that specific server.
 
-Since you will be potentially running multiple FakeFish instances, you will make use of an environment variable to configure in which port a given FakeFish instance listens on and another one to configure which BMC IP it provides service to. On top of that, you can do a bind mount for the folder containing the scripts for managing that specific server or use the scripts inside the container image.
+Currently, FakeFish exposes the following arguments:
+
+|Argument|What is it used for|Default|Required|
+|--------|-------------------|-------|--------|
+|`--tls-mode`| Configures TLS for FakeFish, three available modes: `adhoc` (FakeFish generated certs), `self-signed` (user provided certs), `disabled` (TLS disabled).|`adhoc`|No|
+|`--managed-endpoint` or `-e`|Defines the BMC IP a FakeFish instance will connect to.|`None`|Yes|
+|`--port` or `-p`|Defines the port a FakeFish instance will listen on.|`9000`|No|
```

Comment:
Maybe `--listen-port` to avoid confusion with the BMC port.

Reply:
Done.